pkg/modules/testdata/cr_auth_proxy_no_redis.yaml

```diff
@@ -77,6 +78,10 @@ spec:
         - name: redis
           redis: redis:6.0.8-alpine
```

Comment:
Should the image be updated to 7.2.4 here as well?

Reply:
Updated the image